cmd/scm_integrations.go

```diff
@@ -0,0 +1,167 @@
+package cmd
+
+import (
+	"net/url"
+
+	"github.com/urfave/cli"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/scm_integrations"
+	"github.com/Scalingo/go-scalingo"
+)
+
+var (
+	scmIntegrationsListCommand = cli.Command{
+		Name:     "scm-integrations",
+		Category: "SCM Integrations",
+		Usage:    "List your SCM integrations",
+		Description: `List all the SCM integrations associated with your account:
+
+	$ scalingo scm-integrations
+
+	# See also commands 'scm-integrations-create', 'scm-integrations-delete', 'scm-integrations-import-keys'`,
+
+		Action: func(c *cli.Context) {
+			err := scm_integrations.List()
+			if err != nil {
+				errorQuit(err)
+			}
+		},
+		BashComplete: func(c *cli.Context) {
+			_ = autocomplete.CmdFlagsAutoComplete(c, "scm-integrations")
+		},
+	}
+
+	scmIntegrationsCreateCommand = cli.Command{
+		Name:     "scm-integrations-create",
+		Category: "SCM Integrations",
+		Flags: []cli.Flag{appFlag,
+			cli.StringFlag{Name: "url", Usage: "URL of the SCM integration"},
+			cli.StringFlag{Name: "token", Usage: "Token of the SCM integration"},
+		},
+		Usage: "Link your Scalingo account with your account on a SCM tool",
+		Description: `Link your Scalingo account with your account on a SCM tool:
```

Comment:
Last review: it would be nice to add a word on the purpose of linking SCM account and Scalingo account. Explaining the user why he would want to link both accounts.